starport/pkg/cosmoserror/error.go

```diff
@@ -1,4 +1,4 @@
-package sdkerror
+package cosmoserror
```

Comment:
We should rename `ErrSPNInternal` to simply `ErrInternal` if the package is not only related to `spn`

Reply:
Done